### PR TITLE
storage: start mtda-usb-functions after storage is loaded

### PR DESCRIPTION
**Pull request by ricbha**
Closes: #148 
Signed-off-by: Richa Bharti <Richa_Bharti@mentor.com>

### CODE REVIEW

**Comment by ricbha** [comment]
@chombourger Should changes be added to https://github.com/siemens/mtda/blob/master/docs/fixtures.rst?

File: meta-isar/recipes-conf/mass-storage/usb-ms-use-sda/10-after-sda.conf [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Copyright (C) 2021 Siemens Digital Industries Software
 # SPDX-License-Identifier: MIT
 [Unit]
+Wants=dev-sda.device
```
Comment (chombourger) [inline meta-isar/recipes-conf/mass-storage/usb-ms-use-sda/10-after-sda.conf:4]: for completeness we should also update the systemd unit for mmcblk0 with the same change

**Comment by chombourger** [comment]
we should also rename `usb-ms-use-sda_0.4.bb` to `usb-ms-use-sda_0.5.bb` and `usb-ms-use-mmcblk0_0.4.bb` to `usb-ms-use-mmcblk0_0.5.bb`

this is needed for the packages on Gemfury to be updated

**Comment by chombourger** [comment]
> @chombourger Should changes be added to https://github.com/siemens/mtda/blob/master/docs/fixtures.rst?

good catch. we should

**Review by chombourger — APPROVED** [review]

**Comment by chombourger** [comment]
Looks good to me, thank you so much for your work!

**Pull request merged**